scss/_variables-dark.scss

```diff
@@ -7,35 +7,46 @@
 //
 
 // scss-docs-start sass-dark-mode-vars
+// scss-docs-start theme-color-dark-variables
+$primary-dark:                      $blue !default;
```

Comment:
You probably want to use `$primary`, `$secondary`, etc. instead of `$blue`, `$gray-600` here in order to keep things synchronized.